src/Tribe/Views/V2/Views/Traits/iCal_Data.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * Provides methods for getting iCal data for views
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Views
+ */
+
+namespace Tribe\Events\Views\V2\Views\Traits;
+
+use Tribe\Events\Views\V2\View;
+
+/**
+ * Trait iCal_Data
+ *
+ * @since   TBD
+ *
+ * @package Tribe\Events\Views\V2\Views\Traits
+ *
+ * @property string $string The slug of the View instance.
+ */
+trait iCal_Data {
+	/**
```

Comment:
:no_entry_sign: Trait name must begin with a capital letter

<sub>hash: 1e61b464e936f4ae9e25b283c6527726</sub>

Reply:
@lucatume do we want to divert here due to it being iCal? Or change to Ical/ICal?

Reply:
I think `iCal` is fine, it's the "official" abbreviation.